packages/suite/src/views/wallet/tokens/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+
+import { WalletLayout } from 'src/components/wallet';
+import { useSelector } from 'src/hooks/suite';
+import { CoinsTable } from './coins/CoinsTable';
+import { Route, Switch } from 'react-router-dom';
+import { TokensNavigation } from './TokensNavigation';
+import { HiddenTokensTable } from './hidden-tokens/HiddenTokensTable';
+
+export const Tokens = () => {
+    const [searchQuery, setSearchQuery] = useState('');
+
+    const { selectedAccount } = useSelector(state => state.wallet);
+
+    if (selectedAccount.status !== 'loaded') {
+        return <WalletLayout title="TR_TOKENS" account={selectedAccount} />;
+    }
+
+    return (
+        <WalletLayout title="TR_TOKENS" account={selectedAccount}>
+            <TokensNavigation
+                selectedAccount={selectedAccount}
+                searchQuery={searchQuery}
+                setSearchQuery={setSearchQuery}
+            />
+            <Switch>
+                <Route path="/accounts/tokens/hidden">
```

Comment:
use `resolveStaticPath` otherwise it will not work

Reply:
resolveStaticPath seems to be used only for static files. I've added an asset prefix to the path and it works on the staging web.

Reply:
Okay, I added the prefix and now it works on the staging web